src/style.css

```diff
@@ -205,11 +214,13 @@ ul {
 #btn__clear-done-tasks {
   height: 3rem;
   padding: 0.5rem 2.5rem;
-  background-color: rgb(238, 237, 237);
-  color: rgb(109, 107, 107);
+  background-color: rgba(19, 77, 162, 0.25);
+  color: #134da2;
 }
 
 .task__header {
+  background-color: #134da2;
+  color: #fff;
   position: relative;
 }
```

Comment:
**✔Project approved ✔**

![image](https://user-images.githubusercontent.com/54015740/149423000-6adf19bd-b671-4f34-8b98-b1dc8f9a3640.png)

You did a very good job, and now is time to merge 😎💯Happy coding! 😄